plugins/catalog-graph/src/components/CatalogGraphCard/CatalogGraphCard.tsx

```diff
@@ -140,6 +141,7 @@ export const CatalogGraphCard = (
         mergeRelations={mergeRelations}
         direction={direction}
         relationPairs={relationPairs}
+        filterFunc={filterFunc}
         zoom={zoom}
       />
     </InfoCard>
```

Comment:
Should we also expose `filterFunc` from `CatalogGraphPage` too?

Reply:
Yes would keep the component interface the same, no matter card or page.